app/controllers/api/v1/users_controller.rb

```diff
@@ -2,9 +2,9 @@
 
 module Api
   module V1
-    class Api::V1::UsersController < Api::ApiController
+    class Api::V1::UsersController < BaseController
```

Comment:
Shouldn't have the Api::V1 prefixing the name inside the namespace. This translates to Api::V1::Api::V1::UsersController, which will confuse the autoloader.

Reply:
good spot. copy paste fail